gluten-delta/src-delta/test/scala/org/apache/gluten/execution/DeltaSuite.scala

```diff
@@ -250,4 +250,65 @@ abstract class DeltaSuite extends WholeStageTransformerSuite {
       }
     }
   }
+
+  testWithSpecifiedSparkVersion(
+    "delta: need to validate delta expression before execution",
+    Some("3.5")) {
```

Comment:
This is a common issue for all Spark version, we need change it to `test(delta: need to validate delta expression before execution")`

Reply:
sure, this is done